lib/routes/thewirehindi/templates/description.art

```diff
@@ -0,0 +1,9 @@
+{{@ excerpt }}
+<br><br>
+{{ if image }}
+<img src="{{ image }}" alt="{{ altText }}" style="max-width: 100%; height: auto;">
+<br><br>
+{{ /if }}
+{{ if content }}
+{{@ content }}
```

Comment:
`excerpt` is a portion of `content`. If both are shown, it means users will see duplicated content.